easybuild/easyblocks/t/trinity.py

```diff
@@ -230,7 +233,7 @@ def build_step(self):
     def install_step(self):
         """Custom install procedure for Trinity."""
 
-        if LooseVersion(self.version) < LooseVersion('2012-10-05'):
+        if LooseVersion(self.version) < LooseVersion('2012-10-05') and LooseVersion(self.version) > "3":
```

Comment:
same here, compare with `LooseVersion('3.0')`

also, add a comment above this line to make it clear that this does not apply for Trinity 2.x (which explains the somewhat weird `> 3` requirement

maybe we should even change the `> 3` to `>= 2012`?

this is kind of dead code though, we don't ship any Trinity easyconfigs older than the 2012-10-05 'version' (but keep it there)